global.json

```diff
@@ -341,6 +341,14 @@
             "repoUrl": "git://github.com/feedhenry-templates/welcome-android.git",
             "githubUrl": "https://github.com/feedhenry-templates/welcome-android.git",
             "repoBranch": "refs/heads/master"
+          },
+          {
+            "id": "welcome_project_windows",
+            "name": "Welcome Project-windows",
+            "type": "client_native_windowsphone8",
```

Comment:
I'm not sure about the type, do we need to introduce a `windows universal` type?